topics/metabolomics/slides/introduction.html

```diff
@@ -0,0 +1,108 @@
+---
+layout: introduction_slides
+logo: "GTN"
+title: "Introduction to Metabolomics"
+type: "introduction"
+contributors:
+  - lecorguille
+  - Cécile Canlet - INRA - UMR Toxalim - France
```

Comment:
Can you add her to the `CONTRIBUTORS.yaml` file? With something like:

```
cecilecanlet:
    name: Cécile Canlet
    github: false
```

And replace here by `cecilecanlet`

Reply:
Cool!
Done